pages/_app.tsx

```diff
@@ -79,9 +80,7 @@ $$    $$ |$$    $$ |$$ |  $$ |$$ |$$ |$$    $$ |$$    $$/ $$    $$/
 
   return (
     <>
-      <Head>
-        <meta name="viewport" content="width=device-width" />
-      </Head>
```

Comment:
this should actually be in the `<Head>` tag in `_document.tsx`, not sure why its not

Reply:
it actually complains when it's in _document which is why i placed it inside _app